rpcs3/Emu/Cell/PPUModule.cpp

```diff
@@ -1307,7 +1320,7 @@ void ppu_load_exec(const ppu_exec_object& elf)
 	ppu->cmd_push({ppu_cmd::initialize, 0});
 
 	// TODO: adjust for liblv2 loading option
-	if (!g_cfg_load_liblv2)
+	if (g_cfg_lib_loader.get() != lib_loader_mode::manual)
```

Comment:
shouldn't this say `if (g_cfg_lib_loader.get() != lib_loader_mode::liblv2only)` ? o.O what a mistake